src/content/factory/space-manager.factory.ts

```diff
@@ -0,0 +1,17 @@
+import {SpaceManager} from "../manager/space.manager";
+
+export class SpaceManagerFactory {
+
+    public createManager(
+        id?: string,
```

Comment:
I don't see the point of these parameters as they aren't provided or used anywhere. Can we just remove them?